sys/src/9/port/devether.c

```diff
@@ -245,12 +258,13 @@ etheroq(Ether* ether, Block* bp)
 		splx(s);
 	}
 
-	if(!loopback){
-		qbwrite(ether->oq, bp);
-		if(ether->transmit != nil)
-			ether->transmit(ether);
-	} else
+	if(loopback){
```

Comment:
geez. Wonder why they did stuff like this? Ah right ... loopback was added much later.